bin/yarn

```diff
@@ -1,2 +1,27 @@
-#!/usr/bin/env node
-require('./yarn.js');
+#!/bin/sh
```

Comment:
This is mostly copied from what npm does, which uses @ForbesLindesay's `cmd-shim`: https://github.com/ForbesLindesay/cmd-shim/blob/master/index.js#L115